filesystem/azure.go

```diff
@@ -212,7 +212,7 @@ func (f *AzureFile) Write(b []byte) (int, error) {
 		return 0, err
 	}
 	if err == io.EOF {
-		return 0, fmt.Errorf("Need blob content")
+		return 0, fmt.Errorf("Need write content")
```

Comment:
Yeah, thanks for reminding me.
I realize that I should check it before, it's my logic problem. I will fix it.
But the old code is correct to achieve this check function, when first read it, if has something, it will not return io.EOF(only read nothing, it return io.EOF).